pom.xml

```diff
@@ -24,7 +24,7 @@
     <parent>
         <groupId>com.github.cafapi</groupId>
         <artifactId>caf-common-parent</artifactId>
-        <version>2.3.0-212</version>
+        <version>3.1.0-SNAPSHOT</version>
```

Comment:
Why would we move to `3.1.0-SNAPSHOT` instead of using the latest released version, i.e. [`3.0.0-318`](https://github.com/CAFapi/caf-common-parent/releases/tag/v3.0.0)?  And does the parent pom version really need to change at all for this?